docs/model/introduction-to-model-architecture.md

```diff
@@ -1,21 +1,21 @@
-# Basics About Model Architecture
+# Introduction to Model Architecture
 
-In this article you will learn about [entities](#entity), [facades](#facade), [repositories](#repository) and their mutual relations.
+In this article you will learn about model, its dependencies, [entities](#entity), [facades](#facade), [repositories](#repository) and their mutual relations.
 
-## Basics about the model
+## Definition of a Model
+Definition of model is adopted form DDD definition of a model. Model is a system of abstractions that describes selected aspect of a domain.
```

Comment:
`a model`
adopted form DDD ???? `adopted from DDD` or `adopted form of DDD` 
DDD could be explained better with some URL